include/ignition/gazebo/components/Visual.hh

```diff
@@ -17,6 +17,9 @@
 #ifndef IGNITION_GAZEBO_COMPONENTS_VISUAL_HH_
 #define IGNITION_GAZEBO_COMPONENTS_VISUAL_HH_
 
+#include <string>
+#include <sdf/Element.hh>
+#include <sdf/parser.hh>
```

Comment:
alphabetize

Reply:
hmm I think this means alphabetize and also taking into account case-sensitivity? If so, done in 9f4a753763f64918d2441b56f16cb8be9040c699